mlflow/protos/webhooks.proto

```diff
@@ -93,14 +93,31 @@ enum WebhookStatus {
   DISABLED = 2;   // Webhook is disabled and not receiving events
 }
 
-// Webhook events enumeration
-enum WebhookEvent {
-  REGISTERED_MODEL_CREATED = 1;
-  MODEL_VERSION_CREATED = 2;
-  MODEL_VERSION_TAG_SET = 3;
-  MODEL_VERSION_TAG_DELETED = 4;
-  MODEL_VERSION_ALIAS_CREATED = 5;
-  MODEL_VERSION_ALIAS_DELETED = 6;
+// Webhook entity types
+enum WebhookEntity {
+  ENTITY_UNSPECIFIED = 0;      // Default/unknown entity
+  REGISTERED_MODEL = 1;        // Registered model entity
+  MODEL_VERSION = 2;           // Model version entity
+  MODEL_VERSION_TAG = 3;       // Model version tag entity
+  MODEL_VERSION_ALIAS = 4;     // Model version alias entity
+}
+
+// Webhook action types
+enum WebhookAction {
+  ACTION_UNSPECIFIED = 0;  // Default/unknown action
+  CREATED = 1;             // Entity was created
+  UPDATED = 2;             // Entity was updated
+  DELETED = 3;             // Entity was deleted
+  SET = 4;                 // Entity was set (e.g., tag set)
```

Comment:
open to discussion: I think having both "created" and "set" is a bit confusing for users. `MODEL_VERSION_TAG.create` or `MODEL_VERSION_ALIAS.set` feels valid combination, but it is not. Do we think we can use `MODEL_VERSION_TAG.create` instead of `MODEL_VERSION_TAG.set`? Otherwise, is there any good way to tell which combination is valid more intuitively?

Reply:
`MODEL_VERSION_TAG.created` would make sense if we had the corresponding endpoint. I think the event should align with the endpoint. `MODEL_VERSION_TAG.create` might lead to confusion.